Using CMAKE_INSTALL_PREFIX is unnecesarry and causes generated target cmake file to contain absolute path.
When us just "bin" CMake generates correct relative paths making resulting package relocatable.

diff --git a/sources/shiboken2/generator/CMakeLists.txt b/sources/shiboken2/generator/CMakeLists.txt
index 1a3f4e5c4..a44489b03 100644
--- a/sources/shiboken2/generator/CMakeLists.txt
+++ b/sources/shiboken2/generator/CMakeLists.txt
@@ -30,7 +30,7 @@ configure_file(shibokenconfig.h.in "${CMAKE_CURRENT_BINARY_DIR}/shibokenconfig.h
 
 install(TARGETS shiboken2
         EXPORT Shiboken2Targets
-        DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
+        DESTINATION "bin")
 
 set(shiboken_generator_package_name "shiboken2_generator")
 
